doc/WebSite/Entity-History.md

```diff
@@ -156,12 +156,12 @@ You can use the **IEntityChangeSetReasonProvider.Use(...)** method as shown belo
     }
 
 The Use method returns an IDisposable and it **must be disposed**. Once the return
-value is disposed, Reason is automatically restored to the previous value.
+value is disposed, the Reason is automatically restored to the previous value.
 
 ### Notes
 
--   A property must be **public** in order to be saved in change logs.
+-   A property must be **public** in order to be saved in the change logs.
     Private and protected properties are ignored.
--   DisableAuditing takes priority over Audited attribute.
--   Only works for entities.
--   Only works for scalar properties, e.g. string, int, bool...
+-   DisableAuditing takes priority over the Audited attribute.
+-   This only works for entities.
+-   This only works for scalar properties, e.g. string, int, bool...
```

Comment:
Again, "the Audited **attribute**" reads well.

It's not obvious what "This" refers to. Take a guess and see if we understand it the same. Maybe better without.

Reply:
Yeah, this is hard to work with.  It does not read well either way.

Reply:
How about "Entity History only works.."

Reply:
Great, we understand it the same :)

I'd rather imply "Entity History" than repeat "Entity History only works".
It's point-form and in the Notes section!